lib/compilers/glimmer/index.js

```diff
@@ -42,3 +57,26 @@ export function getTemplateSpecifier(rootName, relativePath) {
 
   return `template:/${rootName}/${pathParts.join('/')}`;
 }
+
+function getManagerIdFromPath(path) {
+  let parts = path.split('.');
+  let managerId = null;
+
+  if (parts.length > 2) {
+    let lastIndex = parts.length - 1;
+    managerId = parts.splice(lastIndex - 1, 1)[0];
```

Comment:
I can confirm that it breaks things having a `.` in the filename. If that is desired we need to fix some other stuff.